app/templates/user_documentation.html

```diff
@@ -7,29 +7,97 @@ <h1 class="header">Help Menu</h1>
+			</div>
+		</div>
+		<div class="help-div">
+			<button type="button" class="help-title titleHead" onclick="handleHelpTitleClick(this)">How do I select the time I want the attributes clustered at?</button>
+			<div class="help-response">
+				<p>
+					You can select the time you would like the attributes to be clustered at by typing a whole number between 0 and 3000
+					 in the &ldquo;time&rdquo; box that is directly below the proximity attribute table.
+				</p>
+				<p class = "helpP"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 249.00px; height: 114.00px;"><img alt="" src={% static "images/image3.png"%} style="width: 249.00px; height: 114.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
+			</div>
+		</div>
+		<div class="help-div">
+			<button type="button" class="help-title titleHead" onclick="handleHelpTitleClick(this)">How do I select the number of clusters?</button>
+			<div class="help-response">
+				<p>
+					You can select the amount of clusters you would like to see in the graph by typing a whole number between 1 and 20 in the 
+					“Cluster” box that is directly below the diversity attribute table.
+				</p>
+				<p class = "helpP"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 267.00px; height: 103.00px;"><img alt="" src={% static "images/image6.png"%} style="width: 267.00px; height: 103.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
+			</div>
+		</div>
+		<div class="help-div">
+			<button type="button" class="help-title titleHead" onclick="handleHelpTitleClick(this)">How do I view the chart generated by the clustering algorithm?</button>
+			<div class="help-response">
+				<p>
+					You can view the chart generated by the clustering algorithm by clicking on the green &ldquo;Process&rdquo; button and 
+					then scrolling down to the Chart section.
+				</p>
+				<p class = "helpP"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 440.00px;"><img alt="" src={% static "images/image4.png"%} style="width: 624.00px; height: 440.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
+			</div>
+		</div>
+		<script src="{% static 'help.js' %}"></script>
```

Comment:
Put this inside a block. In base.html, replace the script with {% block scripts %}{% endblock %} and in user_documentation.html and index.html, include the necessary scripts in the block.